.github/workflows/translations.yml

```
@@ -2,7 +2,7 @@ name: Translations
 on:
   push:
     branches:
-      - master
+      - 6.6.x
```

```suggestion
      - ${{ vars.RELEASE_BRANCH }}
```
Should be this, shouldn't it?

---

I don't think so 😄